domains/certificates/RegistryConnector/Worker/ProjectOriginOptions.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using Microsoft.Extensions.DependencyInjection;
+using ProjectOrigin.HierarchicalDeterministicKeys.Interfaces;
+
+namespace RegistryConnector.Worker;
+
+public class ProjectOriginOptions
+{
+    public const string ProjectOrigin = nameof(ProjectOrigin);
+
+    [Required]
+    public string RegistryUrl { get; set; } = "";
+    [Required]
+    public string RegistryName { get; set; } = "";
+    public byte[] Dk1IssuerPrivateKeyPem { get; set; } = Array.Empty<byte>();
+    public byte[] Dk2IssuerPrivateKeyPem { get; set; } = Array.Empty<byte>();
```

Comment:
It is to avoid warnings about a non-nullable property being uninitialized. 
The validation logic (which not alone checks there is a value, but also that it is an actual private key) is executed on startup - the server will not start if the validation check fails. So they will always be in a valid state.